### PR TITLE
Structuring about page and providing information of all the contributors

### LINKED ISSUE #220: Lower down the hierarchy of the about button
The about button is very prominently shown and it should be one of the things with a comparatively lesser priority in the list of interactive elements.

### LINKED ISSUE #240: Structuring about page and providing information of all the contributors
About page should be made by describing mote project and then providing a list of all the contributors who have contributed to the project by giving info about their name, profile picture and the link to their github profile

### PR DESCRIPTION
**Pull request by subhangi2731**
About page is made by describing mote project and its aim and then providing a list of all the contributors who have contributed to the project by giving information about their name, profile picture, and the link to their GitHub profile

closes #240 
closes #220

### CODE REVIEW

**Comment by subhangi2731** [comment]
Screenshot from 2022-02-28 12-21-39

**Comment by subhangi2731** [comment]
Screenshot from 2022-02-28 12-32-33
Screenshot from 2022-02-28 12-32-54

**Comment by subhangi2731** [comment]
@t0xic0der @darknao let me know if any more changes are required for about page

File: mote/templates/aboutpage.html [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <link href="{{ url_for('static', filename='css3/bootstrap.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/custom_fonts.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fragment.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fontawesome-all.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fullcalendar.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/about.css') }}" rel="stylesheet"/>
```
Comment (darknao) [inline mote/templates/aboutpage.html:31]: Please don't create a new css only for this page. Use the existing one `fragment.css`.

**Review by darknao — CHANGES_REQUESTED** [review]
Quick review, only focused on the contributors list:

**Comment by subhangi2731** [comment]
Screenshot from 2022-03-01 19-31-49
Screenshot from 2022-03-01 19-32-06

**Comment by subhangi2731** [comment]
@t0xic0der @darknao I have done the changes. Please check and let me know if everything is okay

File: mote/templates/aboutpage.html [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+                            The frontend is written in HTML5, CSS3 and ES6 with the use of libraries such as Bootstrap 5, jQuery, Popper.JS, EasyQRCode etc
+                            </p>                    
+                            <h2 class="abouthead"><u>List of contributors for mote project</u></h2>
+                            <div class="row row-cols-1 row-cols-sm-2  row-cols-lg-3 row-cols-xl-4 g-4">
+                            <div class="col">
+                              <div class="card border-light bg-light mb-3">
+                                <div class="row g-0">
+                                  <div class="col-4">
+                                    <a href="https://github.com/cydrobolt">
+                                      <img src= "https://avatars.githubusercontent.com/u/1058150?v=4" style="width: 100px; height: 100px;" class="rounded-circle" alt="Chaoyi Zha">
```
Comment (darknao) [inline mote/templates/aboutpage.html:111]: This image use the `rounded-circle` class while the others use `rounded-start`.
For consistency, you should choose one or another. Both looks good, so it's up to you to decide.

File: mote/templates/aboutpage.html [hunk 3]
```diff
@@ -0,0 +1,743 @@
...
+
+                            <h4>Flask project used to aggregate and distribute IRC meeting minutes and logs for the Fedora Project</h4>
+                        
+                             <h2 class="abouthead"><u>Description</u></h2>
+                
+                            <p>
+                            Mote is a web application purposed to aggregate and distribute the minutes and logs for IRC/Matrix meetings of the Fedora Project community
+                            The project allows looking up meeting info with channel names, dates/times of occurrence and lists recent meetings for quick access
+                            The backend is written in Python with the use of libraries such as Flask micro web framework, BeautifulSoup4, Urllib3, Click, Werkzeug etc
+                            The frontend is written in HTML5, CSS3 and ES6 with the use of libraries such as Bootstrap 5, jQuery, Popper.JS, EasyQRCode etc
```
Comment (darknao) [inline mote/templates/aboutpage.html:102]: Add punctuation at the end of sentences. This paragraph is a bit hard to read without it.
Maybe some proper line breaks would help too.

File: mote/templates/aboutpage.html (hunk 2 above)
Comment (subhangi2731) [reply mote/templates/aboutpage.html:111]: sure

File: mote/templates/aboutpage.html (hunk 3 above)
Comment (subhangi2731) [reply mote/templates/aboutpage.html:102]: sure

**Comment by subhangi2731** [comment]
@t0xic0der @darknao I have done all the changes. Please check and let me know if everything is okay and ready to be merged?

File: mote/templates/aboutpage.html [hunk 4]
```diff
@@ -0,0 +1,746 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <link href="{{ url_for('static', filename='css3/bootstrap.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/custom_fonts.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fragment.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fontawesome-all.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fullcalendar.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/about.css') }}" rel="stylesheet"/>
```
Comment (darknao) [inline mote/templates/aboutpage.html:31]: This file should not be here anymore

File: mote/templates/aboutpage.html [hunk 5]
```diff
@@ -0,0 +1,746 @@
...
+                                    </div>
+                                  </div>
+                                </div>
+                                  </div>
+                                </div>
+                                <div class="col">
+                                  <div class="card border-light bg-light mb-3">
+                                    <div class="row g-0">
+                                    <div class="col-4">
+                                    <a href="https://github.com/vivekanand1101"> -->
```
Comment (darknao) [inline mote/templates/aboutpage.html:385]:
```suggestion
                                    <a href="https://github.com/vivekanand1101">
```

File: mote/templates/aboutpage.html [hunk 6]
```diff
@@ -0,0 +1,746 @@
...
+        <script src="{{ url_for('static', filename='jscn/fragment.js') }}"></script>
+        <script src="{{ url_for('static', filename='jscn/easy.qrcode.min.js') }}"></script>
+        <script src="{{ url_for('static', filename='jscn/socket.io.js') }}"></script>
+        <script src="{{ url_for('static', filename='jscn/fullcalendar.js') }}"></script>
+        <title>Meetbot Logs</title>
+    </head>
+    <body class="body bg-light">
+        <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light" style="background-image: linear-gradient(180deg,#eee 0,#ddd);">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#" style="width: 175px; margin-right: 0px;">
```
Comment (darknao) [inline mote/templates/aboutpage.html:44]: Add a link back to the home page here.
Also, the "Recent meetings" menu item should also link back to the home page.

**Review by darknao — CHANGES_REQUESTED** [review]
Few last remarks:

**Comment by subhangi2731** [comment]
@darknao @t0xic0der I have done all the changes. Is this ready to be merged?

**Review by darknao — APPROVED** [review]
LGTM

**Comment by t0xic0der** [comment]
@subhangi2731 I am trying to access path `/about` and I do not seem to be getting the page.

All I get is this. 

image

I tried clicking on the `About` link on the navbar but it directs to `#`. 

How do I access the page?

**Comment by subhangi2731** [comment]
@t0xic0der ohh let me check and update you on this

**Comment by darknao** [comment]
looks fine to me:
image

**Comment by t0xic0der** [comment]
Let me purge my local repo, clone again and give it a try.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Right, I have some changes that I wanna suggest.

image

Please remove the underline from all the headers.

Add some vertical spaces between the sections of "Mote 2.0", "Description" and "List of contributors for mote project".

The paragraph under the "Description" section is apparently not a paragraph but a group of four lines. Please make it a paragraph. 

image

I am wondering if the `About` page should be an `About` modal with just the actual header "Mote 2.0", the "Description" section and a link which leads to https://github.com/fedora-infra/mote/graphs/contributors. 

Opinions, @darknao @mairin?

**Comment by subhangi2731** [comment]
@t0xic0der @darknao I did all the changes as said above. Is this ready to be merged?

**Review by t0xic0der — APPROVED** [review]
LGTM.

Merging.

**Pull request merged**